crypto/threads_pthread.c

```diff
@@ -453,7 +453,7 @@ static struct rcu_qp *update_qp(CRYPTO_RCU_LOCK *lock)
     /*
      * update the reader index to be the prior qp
```

Comment:
If we fixing this comment, should we fix the capitalisation/punctuation in this phrase?

```
 * Update the reader index to be the prior qp.
```

Reply:
One-liners often lack uppercasing and period, but yes, I can fix that.